packages/manager/apps/telecom/src/app/telecom/telephony/line/phone/order/choice/telephony-line-phone-order-choice.html

```diff
@@ -0,0 +1,180 @@
+<div>
+    <header class="widget-presentation-header">
+        <h2
+            class="widget-presentation-title"
+            data-translate="telephony_line_phone_order_order_title"
+        ></h2>
+    </header>
+
+    <div class="text-center" data-ng-if="$ctrl.isStepLoading">
+        <oui-spinner></oui-spinner>
+    </div>
+
+    <div data-ng-if="!$ctrl.isStepLoading">
+        <div
+            data-ng-if="($ctrl.phoneOffers.length > 1 || $ctrl.merchandise.length)"
+        >
+            <!-- Define filters -->
+            <oui-dropdown class="d-flex justify-content-end">
+                <oui-dropdown-trigger
+                    text="{{ ::'telephony_line_phone_filter_price' | translate }}"
+                >
+                </oui-dropdown-trigger>
+                <oui-dropdown-content>
+                    <oui-dropdown-item data-on-click="$ctrl.sortPriceAsc()">
+                        <span
+                            data-translate="telephony_line_phone_filter_price_asc"
+                        ></span>
+                    </oui-dropdown-item>
+                    <oui-dropdown-item data-on-click="$ctrl.sortPriceDesc()">
+                        <span
+                            data-translate="telephony_line_phone_filter_price_desc"
+                        ></span>
+                    </oui-dropdown-item>
+                </oui-dropdown-content>
+            </oui-dropdown>
+
+            <!-- Define tabs -->
+            <oui-tabs>
+                <oui-tabs-item
+                    data-ng-repeat="brand in $ctrl.brandList"
```

Comment:
```suggestion
                    data-ng-repeat="brand in $ctrl.brandList track by brand.id"
```
Or something else (`$index`if no other solution), to ensure unique values